umei-openapi.json

```diff
@@ -1488,6 +1477,105 @@
           "type": "string",
           "nullable": true
         }
+      },
+      "timeStamp": {
+        "name": "timeStamp",
+        "description": "A timestamp, with a resolution of seconds, in ISO-8601 format with Z (preferred), timezone identifier or offset",
+        "in": "query",
+        "required": false,
+        "schema": {
+          "type": "string"
+        }
+      },
+      "periodFrom": {
+        "name": "periodFrom",
+        "description": "Specify the \"periodFrom\" by exact match (with a resolution of one second)",
+        "in": "path",
```

Comment:
Isn't it a query parameter instead of a path parameter ?
+ Idem for the other parameters below